openmetadata-ui/src/main/resources/ui/src/components/Settings/Users/users.less

```diff
@@ -501,3 +501,16 @@
   white-space: nowrap;
   max-width: 140px;
 }
+.inherit-icon-container {
+  background-color: @white;
+  border-color: @primary-color;
+  color: @primary-color;
+  border: 1px solid @primary-color;
+  width: @size-sm;
+  height: @size-sm;
```

Comment:
Let's not use size variable for height and width as it's mostly use for spacing purpose. If we change spacing globally this will also get affected with that.

Reply:
understood, updated.